File: dotnet/GitHub/Repos/Item/Item/Pulls/Item/Reviews/Item/Events/EventsPostRequestBody_event.cs
namespace GitHubTodoDemo.GitHub.Repos.Item.Item.Pulls.Item.Reviews.Item.Events {
    /// <summary>The review action you want to perform. The review actions include: `APPROVE`, `REQUEST_CHANGES`, or `COMMENT`. When you leave this blank, the API returns _HTTP 422 (Unrecognizable entity)_ and sets the review action state to `PENDING`, which means you will need to re-submit the pull request review using a review action.</summary>
    public enum EventsPostRequestBody_event {
        APPROVE,
        REQUEST_CHANGES,
        COMMENT,
    }
}

File: php/src/GitHub/Repos/Item/Item/Pulls/Item/Reviews/Item/Events/EventsPostRequestBody_event.php
<?php

namespace Baywet\Githubtododemo\Github\Repos\Item\Item\Pulls\Item\Reviews\Item\Events;

use Microsoft\Kiota\Abstractions\Enum;

class EventsPostRequestBody_event extends Enum {
    public const A_P_P_R_O_V_E = 'APPROVE';
    public const R_E_Q_U_E_S_T__C_H_A_N_G_E_S = 'REQUEST_CHANGES';
    public const C_O_M_M_E_N_T = 'COMMENT';
}
